lib/streamlit/elements/write.py

```diff
@@ -131,18 +146,13 @@ def write(self, *args, **kwargs):
             height: 200px
 
         """
-        # Import dynamically, to resolve circular dependency
-        from streamlit.delta_generator import DeltaGenerator
-
-        # Cast self to DeltaGenerator, to resolve mypy type issues
-        dg = cast(DeltaGenerator, self)
```

Comment:
Converted to the new `self.dg` convention used in the other mixins

Reply:
❤️